lib/send_recover_link.rb

```diff
@@ -0,0 +1,25 @@
+require 'mailgun'
+
+class SendRecoverLink
+
+  def initialize(mailer: nil)
+    @mailer = mailer || Mailgun::Client.new(ENV['MG_API_KEY'])
+  end
+
+  def self.call(user, mailer = nil)
+    new(mailer: mailer).call(user)
+  end
+
+  def call(user)
+    mailer.send_message(
+      ENV['MG_DOMAIN_NAME'],
+      from: "postmaster@" << ENV['MG_DOMAIN_NAME'],
+      to: user.email,
+      subject: "Reset your password",
+      text: "Click here to reset password: http://ultra-chitter.herokuapp.com/users/reset_password?token=#{user.password_token}")
```

Comment:
Metrics/LineLength: Line is too long. [129/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)